### PR TITLE
task/logging_abstraction/log_lifecycle

### PR DESCRIPTION
**Pull request by BarAgam**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/43" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/view/BaseActivity.java [hunk 1]
```diff
@@ -6,9 +6,19 @@
 import android.support.v7.app.AppCompatActivity;
 
 import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.logging.LogWrapper;
+import com.teamagam.gimelgimel.app.common.logging.LogWrapperFactory;
 
 public abstract class BaseActivity<T extends Application> extends AppCompatActivity {
 
+    private static final LogWrapper sLogger = LogWrapperFactory.create(BaseActivity.class);
```
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/BaseActivity.java]: would BaseActivity<T>.class work?
If not, we have to find a way to determine which activity is currently active. !resolve

File: app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/BaseFragment.java [hunk 2]
```diff
@@ -30,34 +31,78 @@ public BaseFragment() {
     }
 
     @Override
+    public void onAttach(Context context) {
```
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/BaseFragment.java:34]: Line 28 - Maybe this is the solution (Couldn't mark this line - out of the review section).

File: app/src/main/java/com/teamagam/gimelgimel/app/view/BaseActivity.java (hunk 1 above)
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/BaseActivity.java]: !resolved

**Pull request merged**